free-1.md

```diff
@@ -58,11 +74,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I feel like this language would require a lot of work in both domains. The work would require code that is more key word/ natural langauge based which would be alot in themsleves. additionally it would require that each action of a keyword be brought to life through automated functionality within the domain of database set up. 
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+I think this idea is pretty big. It would require students to have to know how data bases work and how to set them up through programing. 
+
```

Comment:
I would also add that you might run into design things about the type of data and the information that it needs - a book's metadata and a note's metadata, and a website's metadata are going to look different, and the ways that a user might want to access and view all of those are going to be different.